### ISSUE TITLE
Malformed HTML in auto-generated page "Quantity"

### ISSUE THREAD

**Issue by lindhe** [kubernetes/website#46476]
It seems like this auto-generated page has some formatting issues: https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/#Quantity

![image](https://github.com/kubernetes/website/assets/7773090/807de3ed-7a2d-4e7a-88a7-137bc73332ac)

I see HTML tags in the code block that I'm guessing should be parsed as HTML not code block.

**Comment by sftim** [comment]
Thanks

I think this is a duplicate of issue https://github.com/kubernetes/website/issues/35712

**Comment by lindhe** [comment]
Indeed. Sorry, I posted quickly and forgot to search for existing issues.
